src/ResourceManager/Network/Commands.Network/ApplicationGateway/BackendHttpSettings/AzureApplicationGatewayBackendHttpSettingsBase.cs

```diff
@@ -78,6 +78,12 @@ public class AzureApplicationGatewayBackendHttpSettingsBase : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public List<PSApplicationGatewayAuthenticationCertificate> AuthenticationCertificates { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Application gateway Trusted Root Certificates")]
+        [ValidateNotNullOrEmpty]
+        public List<PSApplicationGatewayTrustedRootCertificate> TrustedRootCertificates { get; set; }
```

Comment:
why is this a plural? the param name should be singular i.e `TrustedRootCertificate`. [Documentation](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#noun-singularity).